packages/backend-function/src/function_env_type_generator.test.ts

```diff
@@ -57,7 +56,6 @@ void describe('FunctionEnvironmentTypeGenerator', () => {
   });
 
   void it('generated type definition file has valid syntax', async () => {
-    const targetDirectory = await fsp.mkdtemp('func_env_type_gen_test');
```

Comment:
why do we change this test?

Reply:
I discussed with @stocaaro about this and found this is not being used (maybe was used in a previous iteration of this test before we moved to `.amplify/generated/env`)

Reply:
Appreciate the help debugging @rtpascual